### DISCUSSION TITLE
Is there already anything the opposite of WaitElementsMoreThan?

### DISCUSSION THREAD

**Discussion by abakum** [go-rod/rod#878] [Q&A] [answered]
```
func WaitElementsLessThan(p *rod.Page, selector string, num int) error {
	return p.Wait(rod.Eval(`(s, n) => document.querySelectorAll(s).length < n`, selector, num))
}
```
Did I invent a bicycle?

**Comment by ysmood** [comment]
Could you provide a use case?

**Reply by abakum** [reply]
https://github.com/go-rod/rod/pull/880#issuecomment-1566894347

**Reply by Fly-Playgroud** [reply, marked as answer]
I think you need [`Page.WaitStable`](https://github.com/go-rod/rod/blob/92a6a5078e54b3dacd92c19c5e517a799d381824/page.go#LL617C16-L617C26),
you could try this.
